README.md

```diff
@@ -18,3 +18,45 @@ Resume a training run specified by `<config_file>` starting from `<run_id>/<chec
 ```
 python src/ --config conf/train/<config_file>.yml --resume models/<run_id>/<checkpoint_file>
 ```
+
+
+## Extending this repo
+
+We follow the following flow for objects in this repository:
+![image of the flow map](<readme_assets/flow map.png>)
+
+These initialized classes are passed the dictionary parsed directly from yaml as arguments. We recommend reading examples for your respective function class in [conf/](https://github.com/in-context-learning-2024/in-context/blob/main/conf/)
+
+Make sure to initialize your parent class with `super().__init__(...)` in both cases
```

Comment:
to me, reads a bit unclear. 'These initialized classes' and 'in both cases' could maybe be reworded to be more precise. (i.e. which classes and which cases?)

Reply:
resolved by saying `FunctionClass` and `ContextModel`